backend/http_settings.go

```diff
@@ -265,6 +284,20 @@ func parseHTTPSettings(jsonData json.RawMessage, secureJSONData map[string]strin
 		}
 	}
 
+	// secure socks proxy
+	if v, exists := dat["enableSecureSocksProxy"]; exists {
+		s.SecureSocksProxyEnabled = v.(bool)
+	}
+
+	if s.SecureSocksProxyEnabled {
+		if v, exists := dat["secureSocksProxyUsername"]; exists {
+			s.SecureSocksProxyUsername = v.(string)
+		}
+		if v, exists := secureJSONData["secureSocksProxyPassword"]; exists {
+			s.SecureSocksProxyUsername = v
```

Comment:
```suggestion
			s.SecureSocksProxyPass = v.(string)
```

?

Reply:
Since this is coming from the secure json data, that's not needed: https://github.com/grafana/grafana-plugin-sdk-go/pull/667/files/8312a7a04cf2615a0e86482ecb2b0b60954634f2#diff-24238925783ddf63370c6983033e083d37c9674a48a7db5419362150e4e6a40bL49

Reply:
You still need to s/SecureSocksProxyUsername/SecureSocksProxyPass

```suggestion
			s.SecureSocksProxyPass = v
```